Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -5,18 +5,17 @@
 import json
 import re
 from datetime import datetime, timedelta
-
 import dateparser
 import pytz
 import requests
+
 from splunklib import client
 from splunklib import results
 from splunklib.data import Record
-import urllib3
 from splunklib.binding import AuthenticationError, HTTPError, namespace
 
-urllib3.disable_warnings()
 
+OUTPUT_MODE = 'json'  # type of response from splunk-sdk query (json/csv/xml)
```

Comment:
rename to `OUTPUT_MODE_JSON`, to give future maintainers some idea about what it is without having to check its value

Reply:
Done.